clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -9950,6 +9950,8 @@ def warn_ret_stack_addr_ref : Warning<
 def warn_ret_local_temp_addr_ref : Warning<
   "returning %select{address of|reference to}0 local temporary object">,
   InGroup<ReturnStackAddress>;
+def err_ret_local_temp_addr_ref : Error<
+  "returning %select{address of|reference to}0 local temporary object">;
```

Comment:
```suggestion
  warn_ret_local_temp_addr_ref.Text>;
```

(or something like that?  Either way, the messages are intentionally the same, would be nice to just share the text).

Reply:
Agree!